spec/models/case_court_report_spec.rb

```diff
@@ -67,7 +58,7 @@
 
       it "has the following keys [:created_date, :casa_case, :case_contacts, :volunteer]" do
         expected = %i[created_date casa_case case_contacts volunteer]
```

Comment:
I would like to refactor this test to be more clear someday, doesn't have to be in this PR tho